src/tv2-common/inewsConversion/converters/ParseCue.ts

```diff
@@ -667,12 +667,10 @@ function parseLYD(cue: string[]) {
 		lydCue.variant = command[1]
 	}
 
-	if (cue[1]) {
-		if (isTime(cue[1])) {
-			lydCue = { ...lydCue, ...parseTime(cue[1]) }
-		} else if (cue[1].match(/;[x|\d+].[x|\d+]x/i)) {
-			lydCue.adlib = true
-		}
+	if (cue[1] && isTime(cue[1])) {
+		lydCue = { ...lydCue, ...parseTime(cue[1]) }
```

Comment:
Does anyone know if there is a specific reason why we in this case refer to the Danish/Norwegian word for "sound" here? Or could we just change it to `soundCue` rather than `lydCue', because then we should probably get at these when we have the chance.

Reply:
Most likely a leftover from early development of blueprints where the implementing developers were less strict of using only english